public/theme/boltcms-2021/partials/_contributors.twig

```diff
@@ -0,0 +1,19 @@
+{% if owner is not defined %}
+    {% set owner = 'bolt' %}
+{% endif %}
+
+{% if repo is not defined %}
+    {% set repo = 'core' %}
+{% endif %}
+
+<div class="contributers-overview">
+    <h3>
+        Without these contributers it wouldn't be possible
```

Comment:
Nice! Making a note here, I need to update this text :-)

Reply:
Check

Reply:
Check